packages/iov-tendermint-rpc/src/client.spec.ts

```diff
@@ -188,6 +188,197 @@ function kvTestSuite(rpcFactory: () => RpcClient): void {
   });
 }
 
+function websocketTestSuite(rpcFactory: () => RpcClient, adaptor: Adaptor): void {
+  // don't print out WebSocket errors if marked pending
+  kvTestSuite(rpcFactory, adaptor);
+
+  it("can subscribe to block header events", done => {
+    pendingWithoutTendermint();
+
+    // seems that tendermint just guarantees within the last second for timestamp
+    const testStart = ReadonlyDate.now() - 1000;
```

Comment:
Could you try `expect(event.time.getTime()).toBeGreaterThanOrEqual(testStart);` instead of this unintuitive substraction?

Reply:
I see. I am fine with leaving the logic as it is. But moving the `- 1000` from `testStart` into the matcher makes the situation much clearer

Reply:
Nice